src/fides/lib/oauth/scopes.py

```diff
@@ -0,0 +1,164 @@
+AUTHORIZE = "authorize"
```

Comment:
These can move into `src/fides/api/ops/api/v1/scopes_registry.py` or the ones from ops could move here. They should already be close to the same, but got out of sync over time.

Reply:
see above comment, created another issue to further deduplicate